docs/source/configuration/metrics.mdx

```diff
@@ -74,17 +74,23 @@ telemetry:
       timeout: 2s
 ```
 
-## Add custom attributes/labels
+## Adding custom attributes/labels
```

Comment:
I'm not 100% sure I got this part right with my edits, could use extra scrutiny

Reply:
it looks good ! Great job thanks !